plugins/infinispan91/pom.xml

```diff
@@ -0,0 +1,182 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+   xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+   <modelVersion>4.0.0</modelVersion>
+
+   <parent>
+      <groupId>org.radargun</groupId>
+      <artifactId>radargun-plugin-parent</artifactId>
+      <version>3.0.0-SNAPSHOT</version>
+      <relativePath>../pom.xml</relativePath>
+   </parent>
+
+   <artifactId>plugin-infinispan91</artifactId>
+   <name>Infinispan 9.1.x plugin for RadarGun</name>
+
+   <properties>
+      <version.infinispan>${infinispan91.infinispan.version}</version.infinispan>
+      <version.narayanajta>5.0.4.Final</version.narayanajta>
```

Comment:
Why have you chosen this one? Narayana has version 5.7 already...

Reply:
Because this is the version used in Infinispan 9.1.x. I actually need to bump the main version to 9.1.4.Final, so I'll update that.

Reply:
+1 I am not sure why we keep the version separately here. We should just grab the transitive dependencies.